src/server/exotics/home/index.njk

```diff
@@ -0,0 +1,65 @@
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{%
+  extends
+  'layouts/page.njk'
+%}
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <div class="govuk-body" data-testid="app-page-body">
+        <div class="govuk-!-margin-top-0 govuk-!-margin-bottom-8">
+          <h1 class="govuk-heading-l">Move animals under disease controls</h1>
```

Comment:
Use `{{ heading }}` as per convention?  Otherwise the `pageHeading` doesn't mean much in the page model